src/CoolPropLib.cpp

```diff
@@ -568,6 +568,41 @@ EXPORT_CODE void CONVENTION AbstractState_get_mole_fractions(const long handle,
         HandleException(errcode, message_buffer, buffer_length);
     }
 }
+EXPORT_CODE void CONVENTION AbstractState_get_mole_fractions_satState(const long handle, const char* saturated_state, double* fractions,
+                                                                      const long maxN, long* N, long* errcode, char* message_buffer,
+                                                                      const long buffer_length) {
+    *errcode = 0;
+
+    try {
+        shared_ptr<CoolProp::AbstractState>& AS = handle_manager.get(handle);
+        std::vector<double> _fractions;
+        double quality = AS->Q();
+        if (0 <= quality && quality <= 1) {
+            if (strcmp("liquid", saturated_state) == 0) {
```

Comment:
It is bad practice to use ``strcmp`` in C++, please change to normal comparisons (e.g. ``saturated_state == "liquid" `` throughout)

Reply:
I hadn't realized you were doing the string comparison with the C char buffer, anyhow better with the ``std::string``